Updated translation_de.ts

mscheidemann opened the pull request:
Updated German translation file:
- added a lot of missing translations
- corrected spelling errors (such as "Betrie**bsy**stem" etc.)
- changed translations according to translation guidelines for German language ([Gnome](https://wiki.gnome.org/de/UebersetzungsRichtlinien)/Debian) etc.

mscheidemann committed:
Update translation_de.ts

Updated German translation, adding missing translations and correcting spelling errors

mscheidemann commented:
BTW: Is there any possibility to translate the "For recovery mode, hold SHIFT ..." message as well?

procount commented:
Many thanks. Translations merged locally for next release.
It seems I forgot to make the Shift message translatable. I will try to fix that too.

procount commented:
Do you have any proposal for the German translation of "For recovery mode, hold SHIFT, click mouse, or press joystick"?
Maybe I can insert it directly for the next version.

"Halten Sie für den Wiederherstellungsmodus die UMSCHALTTASTE gedrückt, klicken Sie mit der Maus oder drücken Sie den Joystick" seems a little verbose.

mscheidemann commented:
I prefer to avoid "Sie" (formal speech) or "du" (informal speech). Quite often this is leading into a mess in community translations. Some use "Sie", others "du" in the next dialog box, which is absolutely horrible.

My suggestion:
"Für Wiederherstellungsmodus:
UMSCHALT-Taste gedrückt halten bzw.
Maustaste oder Joystick drücken"

In German, we don't have to (necessarily) capitalise "Umschalttaste" (this time without hyphen), it is a quite unique term, not used in many other context than keyboard.

> ... seems a little verbose

Well, yes, German is much "longer" than English, which make things partially very difficult for the limited space in GUIs (reserved for text in dialog boxes / buttons).
Especially when it comes to singular/plural forms like "the selected OS(es)". We have different articles for plural or singular forms, and the singular always have one of 3 genders (male, female, neuter), that needs to be declined separately and differently (Nominative, Genitive, Dative, Accusative), e.g. male: der/des/dem/den, female: die/der/der/die – and for the related adjective in front of the substantive (same case).
So "the selected operating system(s)" becomes "das/die gewählte(n) Betriebssystem(e)", which … 1) is not easy to read and kind of unacceptable in menus … 2) did not even include any verb, which determines the grammar case to use ;-)) (I think French is similar complicated and "long"). 


----
-==== Your original message on Monday, 11. April 2022 at 12:45 ====-


Do you have any proposal for the German translation of "For recovery mode, hold SHIFT, click mouse, or press joystick"?
Maybe I can insert it directly for the next version.
"Halten Sie f�r den Wiederherstellungsmodus die UMSCHALTTASTE gedr�ckt, klicken Sie mit der Maus oder dr�cken Sie den Joystick" seems a little verbose.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

mscheidemann committed:
Update translation_de.ts

mscheidemann committed:
Update translation_de.ts

While testing I noticed 2 terms used as nouns (instead of verbs), which makes them uppercase in German.

mscheidemann committed:
Update translation_de.ts
